problemsets-mat223/linearalbegra.tex

```diff
@@ -4240,6 +4240,22 @@ \section*{Task 2.3: Pat and Jamie}
 			\item $\mat{3\\3\\2}$
 			\item $\mat{1&0&1\\0&1&0\\0&0&1}$.
 		\end{enumerate*}
+		\begin{solution}
+			For each part, we compute the reduced row echelon form of the matrix
+			and simply count the number of pivots.
+			\begin{enumerate}
+				\item $\rref\left(\mat{1&1\\2&2}\right)=\mat{1&1\\0&0}$, and so
+					the rank of this matrix is 1.
+				\item $\rref\left(\mat{1&2\\3&4}\right)=\mat{1&0\\0&1}$, and so
+					the rank of this matrix is 2.
+				\item This matrix is already in reduced row echelon form, and so
+					its rank is 2.
+				\item $\rref\left(\mat{3\\3\\2}\right)=\mat{1\\1\\1}$, and so
```

Comment:
nope

Reply:
Whoops. Let's pretend that one never happened.